script/utils/revapi-clean.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 # How to clean ignores in revapi-config.json files:
 #
-# export MAIN_REVAPI_REPOSITORIES=droolsjbpm-build-bootstrap,appformer,droolsjbpm-knowledge,drools,jbpm,droolsjbpm-integration
-# export BRANCHED_7_REVAPI_REPOSITORIES=optaplanner
-# export REVAPI_REPOSITORIES=$MAIN_REVAPI_REPOSITORIES,$BRANCHED_7_REVAPI_REPOSITORIES
+# export MAIN_REVAPI_REPOSITORIES=droolsjbpm-build-bootstrap,appformer,jbpm,droolsjbpm-integration
+# export BRANCHED_7_REVAPI_REPOSITORIES=droolsjbpm-knowledge,drools,optaplanner
+# export REVAPI_REPOSITORIES=droolsjbpm-build-bootstrap,appformer,droolsjbpm-knowledge,drools,jbpm,droolsjbpm-integration,optaplanner
```

Comment:
@afalhambra FYI this was the easiest way to make it work. As for mvn-all command it has to be in the right order. Another option I considered was to split `MAIN_REVAPI_REPOSITORIES` to `MAIN_REVAPI_REPOSITORIES_1` and `MAIN_REVAPI_REPOSITORIES_2` and then interleave it with the 7.x-based repositories in the `REVAPI_REPOSITORIES`.